web/src/app/page.tsx

```diff
@@ -7,10 +8,11 @@ export const metadata = {
 
 export default function Home() {
   return (
-    <div className="page">
-      <h1 id="title--portal">Welcome to the ARC Services Portal</h1>
-      <p>This portal allows UCL researchers to manage ARC services and tasks.</p>
-
+    <div className="page__content__inner--centered">
+      <div className="title">
+        <h1 className="title__content">Welcome to the ARC Services Portal</h1>
```

Comment:
I can't find a style definition for `title__content`, if classes are not being used they should be removed.